### ISSUE TITLE
A11y: The selected icon is missed with radio button in Windows high contrast model 

### ISSUE THREAD

**Issue by icemanlily** [IBM/carbon-components-react#1850]
===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: 1.1.1 Non-Text Content
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/non_text_content.html
Test Tool Type: Manual

===== DEFECT DESCRIPTION =====
All important contents in a web page should be shown in Window High Contrast model.

How to apply a high-contrast theme in Windows
1> Press left Shift + left Alt + PrtSc (print screen) to open the High Contrast dialog.
2> Select the Go to Ease of Access Center link which opens the Make the computer easier to see page.
3> Select the Choose a High Contrast Theme link to open the Personalization control panel.
4> Select one of the high-contrast themes under Basic and High contrast themes. Recommendation: Use High Contrast #1 which is yellow on black for your test. For a more complete test, consider testing with more than one high contrast theme such as white on black.
5> Close the Personalization control panel.
6> Select Apply and OK buttons to apply the selection. Close the Ease of Access Center window, if still open.

=====STEPS TO REPRODUCE =====

1. Set the high-contrast theme in Windows
2. Go to https://dev.console.test.cloud.ibm.com/unifiedsupport/cases/add
3. Select Technical

===== EXPECTED RESULT =====
All important contents in a web page should be shown in Window High Contrast model.

![image](https://user-images.githubusercontent.com/6970471/52547177-a5ea8a80-2e00-11e9-9d86-1a1dfa901147.png)

**Comment by dakahn** [comment]
@IBM/carbon-designers

**Comment by aagonzales** [comment]
The selected dot inside of the radio button is the same color as the border. I'm not sure why the border would show but not the dot.

**Comment by dakahn** [comment]
This is definitely related to [this issue posted in our React repo](https://github.com/IBM/carbon-elements/issues/345). Something about how Windows High Contrast Mode and by extension Firefox parses SVG's.

**Comment by dakahn** [comment]
@joshblack is working on a fix over in the carbon-elements repo. You can track the work [here](https://github.com/IBM/carbon-elements/pull/388).

**Comment by joshblack** [comment]
Yeah, we should bring this in with the elements integration. I think we can solve this upstream in elements 👍

**Comment by dakahn** [comment]
We've got a fix merged into stable Carbon-Elements. PR [#2104](https://github.com/IBM/carbon-elements/pull/388) @icemanlily
